backend/api/urls.py

```
@@ -2,25 +1,68 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
 from .views import (
     TestAuthView, OnboardingAPIView, SetFirebaseTokenView, 
     UploadVideoView, VideoFeedView, VideoDetailView,
     RecordVideoViewAPI, ToggleVideoLikeAPI, 
-    CreateVideoShareAPI, UserHistoryAPI
+    CreateVideoShareAPI, UserHistoryAPI, UserLikedVideosAPI,
+    VideoSearchView, EvaluationFormViewSet, EvaluationQuestionViewSet,
+    SubmitEvaluationResponseView, UserPointsView
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)